### PR TITLE
(WIP)Change embedder penalty API

### PR DESCRIPTION
**Pull request by Nzteb**

### CODE REVIEW

File: kge/model/kge_model.py [hunk 1]
```diff
@@ -426,19 +426,32 @@ def append_num_parameter(job, trace):
         job.post_epoch_trace_hooks.append(append_num_parameter)
 
     def penalty(self, **kwargs) -> List[Tensor]:
+        # Note: If the subject and object embedder are identical, embeddings may be
+        # penalized twice. This is intended (and necessary, e.g., if the penalty is
+        # weighted).
         if "batch" in kwargs and "triples" in kwargs["batch"]:
             kwargs["batch"]["triples"] = kwargs["batch"]["triples"].to(
```
Comment (rgemulla) [inline kge/model/kge_model.py:433]: Store this in local variable triples for readability.

**Comment by rgemulla** [comment]
Ok #97 is in, please rebase accordingly.

**Comment by Nzteb** [comment]
I have rebased and addressed the comment

**Comment by rgemulla** [comment]
Thanks!

**Pull request merged**